fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -151,11 +151,26 @@ def run
       command :socket_server do |c|
         c.syntax = 'fastlane start_server'
         c.description = 'Starts local socket server and enables only a single local connection'
+        c.option '-s', '--stay_alive', 'Keeps socket server up even after error or disconnects, requires CTRL-C to kill.'
```

Comment:
🕺

Reply:
get it?

Reply:
> Feel the city breakin' and everybody shakin' 🎶

Reply:
![huehuehue](https://user-images.githubusercontent.com/1304821/33776584-b72cc964-dbf6-11e7-8ca5-7e0aab7b4483.gif)